yagb-core/src/emulator/ppu/ppu-base.ts

```diff
@@ -2,6 +2,7 @@ import { Bus, ReadHandler, WriteHandler } from '../bus';
 import { Interrupt, irq } from '../interrupt';
 import { Ppu, ppuMode } from '../ppu';
 
+import { Event } from 'microevent.ts';
```

Comment:
I always happy to see `microevent.ts` in use, but I think in this case we'd be better served with a method called `onHblankStart` which is empty in `PpuBase` and overridden in `PpuGbc` --- less code, and slightly less overhead.